scripts/west_commands/tests/test_pyocd.py

```diff
@@ -84,7 +84,7 @@
                              '-ex', 'monitor halt',
                              '-ex', 'monitor reset',
                              '-ex', 'load']
-DEBUG_DEF_EXPECTED_SERVER = ['pyocd-gdbserver',
+DEBUG_DEF_EXPECTED_SERVER = ['pyocd gdbserver',
```

Comment:
It's one string in the list per argument, so the Python expression should be `'pyocd', 'gdbserver'` instead. Similarly elsewhere.

I ran the tests and here are the failures:

https://gist.github.com/mbolivar/654208a6535999c7748ed9bc26eaf3a7

Reply:
Fixed. Tests are now passing locally.